.travis.yml

```diff
@@ -48,6 +33,7 @@ matrix:
   allow_failures:
   - env: EMBER_TRY_SCENARIO=ember-beta
   - env: EMBER_TRY_SCENARIO=ember-canary
+  - node_js: 'stable'
```

Comment:
Same question as other PRs @juwara0 about whether needed and that I haven't bee doing this and whether should be.